src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {

```

Comment:
Lets add some logic to all these methods.
_In those classes override doWork(), so it will print message that certain machine started its work._